### ISSUE TITLE
Feature request: more options for MoveTo

### ISSUE THREAD

**Issue by Vladimir-csp** [labwc/labwc#1803]
- Support `%`-suffixed values
- Support negative (or `-`-prefixed) values to count from the other side.
- Add `origin` option `{top|bottom|center}-{left|right|center}|center` (default: `top-left` as it is now).
- Add `workarea` option to limit placement to workarea (default: yes)

For example this would move window to the center of workarea:
`<action name="MoveTo" x="50%" y="50%" origin="center" />`

These would move window to the right edge and center vertically:
`<action name="MoveTo" x="-0" y="50%" origin="center-right" />`

This would just move window to the left edge:
`<action name="MoveTo" x="0" origin="center-left" />`

**Comment by Consolatis** [comment]
This use-case might be better served with some `MoveToRegion` action.

**Comment by Vladimir-csp** [comment]
`SnapToRegion`? This makes window snapped and always resizes.

**Comment by Vladimir-csp** [comment]
Also, this can be a more generic solution to #1511/#1784

**Comment by Consolatis** [comment]
> `SnapToRegion`? It makes window snapped and always resizes.

Right, a `MoveToRegion` action would only take the x and y values of the region and move the window there.

**Comment by Vladimir-csp** [comment]
This would not solve "move to bottom right corner (or to the center on the right side) and do nothing else" case, since `MoveTo` and supposed `MoveToRegion` both deal with top-left origins and count from top-left.
